src/contracts/Metadata.contract.js

```diff
@@ -82,12 +82,12 @@ export default class Metadata {
     let createdDate = validatorData.createdDate > 0 ? moment.unix(validatorData.createdDate).format('YYYY-MM-DD') : ''
     let updatedDate = validatorData.updatedDate > 0 ? moment.unix(validatorData.updatedDate).format('YYYY-MM-DD') : ''
     let expirationDate = validatorData.expirationDate > 0 ? moment.unix(validatorData.expirationDate).format('YYYY-MM-DD') : ''
-    if (validatorData.zipcode.length === 4) {
-      validatorData.zipcode = "0" + validatorData.zipcode;
-    }
-    if (validatorData.zipcode === "0") {
-      validatorData.zipcode = '';
-    }
+    //if (validatorData.zipcode.length === 4) {
```

Comment:
@varasev could you remind what is the intention to comment these lines in this PR?

Reply:
@vbaranov These lines worked for the old ValidatorMetadata when `zipcode` had `uint256` type, and we had to add a leading zero for 4-digit zip codes for that reason. Now `zipcode` has `bytes32` type and contains all the necessary digits for zip code including a leading zero, so now these lines don't have any sense.

Reply:
Maybe, let's remove these lines then?

Reply:
Yes, I've removed them: https://github.com/poanetwork/poa-dapps-validators/pull/62/commits/8aa5a6edb25d492d89422062219040db264d3ba1